src/themes/uswitch/theme.json

```diff
@@ -340,6 +340,21 @@
     }
   },
 
+  "cta": {
+    "main": {
+      "borderWidth": 1,
+      "borderStyle": "solid",
+      "borderColor": "light-3",
+      "borderLeftWidth": 10,
+      "borderLeftColor": "#A3BCFF",
```

Comment:
variable

Reply:
See PR description - the colours aren't done in the uswitch theme yet. This can be amended as part of that work